CDH-Studio/UpSkill
# Semantic simple search

## What does this do?
This PR adds the v1 'view' for the simple search page. It does not have any proper logic yet.

## Related Jira Tickets
Waiting for Jira to be set up

## Testing Steps
  - run `npm start`
  - ensure page looks similar to Adobe XD mockup

## Notes
N/A
